node/status_node.go

```
@@ -77,6 +78,14 @@ func (n *StatusNode) GethNode() *node.Node {
 	return n.gethNode
 }
 
+// Server retrieves the currently running P2P network layer.
+func (n *StatusNode) Server() *p2p.Server {
+	n.mu.RLock()
+	defer n.mu.RUnlock()
+
+	return n.gethNode.Server()
```

We should return an error
```go
if !n.isRunning() {
	return ErrNoRunningNode
}
```
if the node is not running. Otherwise, `n.gethNode` is `nil` and this will panic.

Alternatively, we can return `nil` which will be more consistent with `gethNode` which returns `nil` when it is not started.